cmd/dep/godep_importer.go

```diff
@@ -125,11 +125,10 @@ func (g *godepImporter) convert(pr gps.ProjectRoot) (*dep.Manifest, *dep.Lock, e
 				ProjectRoot: gps.ProjectRoot(pkg.ImportPath),
 			}
 			revision := gps.Revision(pkg.Rev)
-			version, err := lookupVersionForRevision(revision, pi, g.sm)
-			if err != nil {
-				warn := errors.Wrapf(err, "Unable to lookup the version represented by %s in %s. Falling back to locking the revision only.", pkg.Rev, pi.ProjectRoot)
-				g.logger.Printf(warn.Error())
-				version = revision
+			version, warning := lookupVersionForLockedProject(pi, nil, revision, g.sm)
```

Comment:
I'd even call this variable `err` since the comment below explains why it was logged and ignored (same thing in glide importer).